grpcgcp/doc.go

```diff
@@ -105,7 +107,7 @@ Usage:
 
 	conn, err := grpc.Dial(
 		target,
-		// Following are channel management options:
+		// Following are connection management options:
```

Comment:
How about, "Register and specify the grpc-gcp load balancer." ? We're not really specifying management options here imo.

Reply:
Done.